### PR TITLE
Update 03-wls-operator.md

### PR DESCRIPTION
**Pull request by Anjali93**

### CODE REVIEW

File: samples/kubernetes/end2end/docs/03-wls-operator.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ We need two images, the WebLogic Server Kubernetes Operator image from [Docker H
 Pull the WebLogic Sever Kubernetes Operator 2.3.0 image.
 ```
 docker login
+for logging into oracle registry use with sso creds use:  docker login container-registry.oracle.com
```
Comment (ddsharpe) [inline samples/kubernetes/end2end/docs/03-wls-operator.md:15]: Each of these lines is executable, and this additional line is not.  This is not the correct location for this comment.

**Review by ddsharpe — CHANGES_REQUESTED** [review]

**Comment by ddsharpe** [comment]
@Anjali93 This commit needs to be signed before it can be accepted.  Please review the document at https://github.com/oracle/weblogic-monitoring-exporter/blob/master/CONTRIBUTING.md

**Comment by robertpatrick** [comment]
Is this PR still relevant?  If so, please address the change requested.  If not, please close it.  Please note that we will have to close this PR if there continues to be no response.

**Comment by ddsharpe** [comment]
Closing due to lack of response.  Please reopen or re-file if still desired.

**Pull request closed without merging**